Microsoft.DotNet.Interactive.Jupyter/CompleteRequestHandler.cs

```diff
@@ -17,7 +18,7 @@ public class CompleteRequestHandler: RequestHandlerBase<CompleteRequest>
         private static readonly Regex _lastToken = new Regex(@"(?<lastToken>\S+)$", RegexOptions.Compiled | RegexOptions.IgnoreCase | RegexOptions.CultureInvariant | RegexOptions.Multiline);
 
 
-        public CompleteRequestHandler(IKernel kernel) : base(kernel)
+        public CompleteRequestHandler(IKernel kernel, IScheduler messagePump) : base(kernel, messagePump)
```

Comment:
Is it a scheduler or a message pump? Typically the naming will make more sense if you just call it `scheduler`.